contentcuration/contentcuration/utils/publish.py

```diff
@@ -673,18 +707,31 @@ def process_assessment_metadata(ccnode, kolibrinode):
         }
     )
 
+    return exercise_data
+
+
+def create_kolibri_assessment_metadata(ccnode, kolibrinode):
```

Comment:
Small refactor to move this after file generation and make it conditional depending on the file type.